notebooks/vertex_genai/solutions/gen_ai_evaluation_service.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Please move the import to the top cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done</p>